packages/devtools_app/lib/src/flutter/notifications.dart

```diff
@@ -9,8 +9,7 @@ import 'package:flutter/scheduler.dart';
 
 import 'common_widgets.dart';
 
-const _notificationHeight = 160.0;
-final _notificationWidth = _notificationHeight * goldenRatio;
+final _notificationWidth = 160.0 * goldenRatio;
```

Comment:
maybe just make this a constant as the golden ratio doesn't really make sense now that we aren't fixing the width-height ration?

Reply:
I may re-add `_notificationHeight` back in just to make the constants more semantic.